tools/twix/src/main.rs

```diff
@@ -316,6 +316,12 @@ impl App for TwixApp {
                 self.dock_state.push_to_focused_leaf(tab.into());
             }
 
+            if ui.input_mut(|input| input.consume_key(Modifiers::CTRL, Key::D)) {
+                let (_, tab) = self.dock_state.find_active_focused().unwrap();
```

Comment:
I no panel is focused (don't know if this is possible at the moment), twix would crash. Maybe instead do nothing if no panel is focused

Reply:
That used to be possible but since the most recent `egui_dock` update, twix already crashes on `main` when there is no focused tab, e.g. by closing the last tab. 😉